services/api/pyproject.toml

```diff
@@ -7,9 +7,11 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 environs = "^9.5.0"
+hffs = {git = "https://github.com/huggingface/hffs.git"}
```

Comment:
feel free to specify a commit sha until we do a proper release ;)

Reply:
Done, thanks!